CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing
+
+Thank you for your interest in contributing to the 1Password load-secrets-action project 👋! Before you start, please take a moment to read through this guide to understand our contribution process.
+
+## Testing
+
+After following the steps below for signing commits, you can test against your PR with these steps:
+
+1. Create or use an existing repo to run the `load-secrets` GitHub Action
+2. In a workflow yaml file that uses the GitHub Action, modify the `uses: 1Password/load-secrets-action` line to be
+
+   ```
+   uses: 1Password/load-secrets-action@<branch-name>
+
+   # OR
+
+   uses: 1Password/load-secrets-action@<commit-hash>
+   ```
```

Comment:
`[nitpick]`

Would it be better to separate these into two code blocks that can be copied?

```yaml
uses: 1Password/load-secrets-action@<branch-name>
```

or

```yaml
uses: 1Password/load-secrets-action@<commit-hash>
```

Also recommend using the `yaml` tag to the code snippet so that the syntax is properly highlighted: 
```
   ```yaml
```